octoprint_mrbeam/iobeam/temperature_manager.py

```diff
@@ -388,8 +388,18 @@ def _check_temp_val(self):
                 and self.temperature < self.temperature_max - self.HYTERESIS_TEMPERATURE
             ):
                 self._logger.warn(
-                    "Cooling break duration passed: %ss - Current temp: %s",
+                    "Cooling break duration(%ss) passed: %ss - Current temp: %s",
                     self.cooling_duration,
+                    self.cooling_since,
+                    self.temperature,
+                )
+                self.cooling_resume()
+            elif (
+                self.cooling_tigger_temperature
+                and self.cooling_tigger_temperature < self.temperature_max
+            ):
+                self._logger.info(
+                    "Cooling resume as valid temperature received again: %ss - Current temp: %s",
```

Comment:
I think there is a value missing for this string?

Reply:
You are right 😅